iop/usb/usbmass_bd/src/usb_mass.c

```diff
@@ -705,6 +710,9 @@ static void usb_mass_release(mass_dev *dev)
     dev->bulkEpO   = -1;
     dev->controlEp = -1;
     dev->status    = 0;
+
+    if (devices_initialising > 0) // should handle all errors?
+        devices_initialising--;
 }
 
 static int usb_mass_disconnect(int devId)
```

Comment:
not sure, but is the devId now pointless as we anyway will go through all devIDs?

Reply:
No I don’t think so but this devices_initialising flag could probably done better with a dev->status flag rather than how I’m doing it currently, just waiting to hear if this method is reasonable or not before I spend more time on it